<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSE396_Group1_Interface</class>
 <widget class="QMainWindow" name="CSE396_Group1_Interface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CSE396_Group1_Interface</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCustomPlot" name="myPlot" native="true">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>0</y>
      <width>400</width>
      <height>350</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="deadmanLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>210</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Deadman switch Connected:</string>
    </property>
   </widget>
   <widget class="QLabel" name="hardwareEdit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>390</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="k1Label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="text">
     <string>K1:</string>
    </property>
   </widget>
   <widget class="QLabel" name="hardwareLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>Hardware Connected:</string>
    </property>
   </widget>
   <widget class="QLabel" name="deadmanEdit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>430</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="k2Label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="text">
     <string>K2:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="k1Spin">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>470</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>40.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="k2Spin">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>510</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>40.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="initialLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Initial Values</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="stateLabel">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>390</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>State</string>
    </property>
   </widget>
   <widget class="QLabel" name="stateEdit">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>440</y>
      <width>400</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>System is not ready</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="myPlot2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>400</width>
      <height>350</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>350</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Motor Encoder</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>350</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Encoder</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
